index.js

```diff
@@ -5,13 +5,10 @@ const swaggerUI = require('swagger-ui-express');
 const swaggerDoc = require('./swagger.json');
 const { PokemonController } = require('./controller/controller');
 
-const pokemonController = new PokemonController(new PokemonService());
+const pokemonRouter = require('./routes/pokemonRoutes');
 
 app.use(cors());
-
 app.use(swaggerUI.serve);
-app.get('/', swaggerUI.setup(swaggerDoc));
-
-app.get('/pokemons', async (req,res) => pokemonController.getAllPokemons(req,res));
+app.use('/', pokemonRouter);
```

Comment:
If we break the routing logic then, one approach we can take is breaking it into the `pokemons` route and `moves` route. 
and route `/` route would be used for swagger. 

```suggestion
app.use('/pokemons', pokemonRouter);
```